File: Python Word examples/watermark/get_text_watermark_properties.py
import spirecloudword
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.watermarks_api.WatermarksApi(configuration)

filename = "TextWatermark.doc"
result = api.get_text_watermark_properties(filename, folder="input")

File: PHP Word Examples/WatermarksApiDemo/getTextWatermarkProperties.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\WatermarksApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new WatermarksApi($configuration);

$name = "TextWatermark.doc";
$folder = "input";
$storage = null;
$password = null;
$result = $apiInstance->getTextWatermarkProperties($name, $folder, $storage, $password);